libraries/src/AWS.Lambda.Powertools.Logging/Internal/PowertoolsLogger.cs

```diff
@@ -65,7 +65,7 @@ internal sealed class PowertoolsLogger : ILogger
     ///     Initializes a new instance of the <see cref="PowertoolsLogger" /> class.
     /// </summary>
     /// <param name="name">The name.</param>
-    /// <param name="powertoolsConfigurations">The Powertools configurations.</param>
+    /// <param name="powertoolsConfigurations">The Powertools for AWS Lambda (.NET) configurations.</param>
```

Comment:
I wouldn't change here. comment in code

Reply:
We're unable to use "Powertools" as a standalone brand any more due to internal issues